service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java

```diff
@@ -75,17 +112,11 @@ public ResponseEntity<RunStateResponse> postRun(String workflowUrl, Object workf
           null,
           null);
 
-      result =
-          new ResponseEntity<>(
-              new RunStateResponse().runId(runId).state(RunState.QUEUED), HttpStatus.CREATED);
+      return new ResponseEntity<>(
+          new RunStateResponse().runId(runId).state(RunState.QUEUED), HttpStatus.CREATED);
     } catch (cromwell.client.ApiException e) {
-      System.out.println(e);
-      result =
-          new ResponseEntity<>(
-              new RunStateResponse().runId(runId).state(RunState.SYSTEM_ERROR),
-              HttpStatus.INTERNAL_SERVER_ERROR);
+      System.out.println(e.getMessage());
```

Comment:
We should log the full error even if we only return the short message to the user